stubs/Jetson.GPIO/@tests/stubtest_allowlist.txt

```diff
@@ -1,10 +1,13 @@
 # The high level module can only be imported on a Jetson SBC
 
-Jetson.GPIO
 # error: Jetson.GPIO failed to import. RuntimeError: The current user does not have permissions set to access the library functionalites. Please configure permissions or use the root user to run this. It is also possible that /dev/gpiochip0 does not exist. Please check if that file is present.
+Jetson.GPIO
 
-Jetson.GPIO.gpio
 # error: Jetson.GPIO.gpio failed to import. RuntimeError: The current user does not have permissions set to access the library functionalites. Please configure permissions or use the root user to run this. It is also possible that /dev/gpiochip0 does not exist. Please check if that file is present.
+Jetson.GPIO.gpio
+
+# error: Jetson.GPIO.gpio_pinmux_lookup failed to import. RuntimeError: The current user does not have permissions set to access the library functionalites. Please configure permissions or use the root user to run this. It is also possible that /dev/gpiochip0 does not exist. Please check if that file is present.
+Jetson.GPIO.gpio_pinmux_lookup
```

Comment:
I think we should just remove the duplicate comments and maybe add a more generic comment. Something along the lines that stubtest doesn't work correctly in CI.